src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot;
+ import java.security.PublicKey;
+
+
+public final class Constants {
+    
+    private Constants() {
+
+    }
+
+    public static final class Wrist {
+        public static final int CONVERSION_FACTOR = 360;
+        public static final int PIVOT_MOTOR_ID = 1;
+        public static final double HIGH = 225;
+        public static final double FLAT = 180;
+        public static final double LOW = 135;
+        public static final double WRIST_P = 2;
+        public static final double WRIST_I = 1;
```

Comment:
We hardly ever use I, for a basline PID you should probably use 1, 0, 0 or 0.1, 0, 0